packages/core/admin/package.json

```diff
@@ -160,5 +161,10 @@
         ]
       }
     }
-  }
+  },
+  "browserslist": [
+    "last 3 major versions",
+    "Firefox ESR",
+    "last 2 Opera versions"
+  ]
```

Comment:
I'd prefer if we could move this into a separate file `.browserslist` to have more visibility and to not overload the package.json so much. Wdyt?

Reply:
I have no strong opinions on this, but it needs to be shipped in the NPM package, looking at `npmignore` it _should be_ but I have more confidence in it being shipped in the `package.json` lol